.travis.yml

```diff
@@ -13,7 +13,7 @@ arch:
 
 julia:
   - 1.0
-  - 1.3
```

Comment:
unintended change?

Reply:
I replaced it with `1`, which expands to the latest 1.x. So when new Julia versions are released, we won't need to update this file. Unless we want to test 1.3 specifically, but I don't see much reason to.

Reply:
Oh, I didn't know that! Useful!